unity-renderer/Assets/DCLPlugins/ECS7/ECSComponents/Raycast/RaycastResult.cs

```diff
@@ -0,0 +1,31 @@
+﻿using DCL;
+using DCL.ECSComponents;
+using DCL.ECSRuntime;
+using DCLPlugins.ECS7.ECSComponents.RaycastResult.Serializer;
+using DCLPlugins.ECSComponents.RaycastResult;
+
+namespace DCLPlugins.ECSComponents
+{
+    public class RaycastResultRegister
+    {
+        private readonly ECSComponentsFactory factory;
+        private readonly IECSComponentWriter componentWriter;
+        private int componentId;
+        
+        public RaycastResultRegister(int componentId, ECSComponentsFactory factory, IECSComponentWriter componentWriter)
+        {
+            factory.AddOrReplaceComponent(componentId, RaycastResultSerializer.Deserialize, () => new RaycastResultComponentHandler(componentWriter, DataStore.i.ecs7));
```

Comment:
```suggestion
            factory.AddOrReplaceComponent(componentId, RaycastResultSerializer.Deserialize, null);
```
I see the handler `RaycastResultComponentHandler` it's actually doing nothing.
If no handler is required for the component you could just `null` it

Reply:
done